sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -204,11 +205,14 @@ def get_outputs(op_ix):
         else:
           # Otherwise serialize the source and execute it there.
           # TODO: Use SDFs with an initial impulse.
+          # The Java runner harness strips the base64 encoding. do the same
```

Comment:
`The Java runner harness strips the base64` -> `The Dataflow runner harness strips the base64`

Lack of a ParDoPayload/SourcePayload/... was why the stripping -> ByteValue conversion was done.